agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AzureMonitorLogProcessor.java

```diff
@@ -21,9 +23,16 @@ public void onEmit(Context context, ReadWriteLogRecord logRecord) {
     }
 
     ReadableSpan readableSpan = (ReadableSpan) currentSpan;
-
-    logRecord.setAttribute(
-        AiSemanticAttributes.OPERATION_NAME, OperationNames.getOperationName(readableSpan));
+    if (ConfigurationBuilder.inAzureFunctionsWorker()) {
+      AzureFunctionsCustomDimensions customDimensions =
+          AzureFunctionsCustomDimensions.fromContext(context);
+      if (customDimensions != null) {
+        logRecord.setAttribute(AiSemanticAttributes.OPERATION_NAME, customDimensions.operationName);
+      }
+    } else {
+      logRecord.setAttribute(
+          AiSemanticAttributes.OPERATION_NAME, OperationNames.getOperationName(readableSpan));
+    }
```

Comment:
oh, I don't think we need this code, since `AzureFunctionsLogProcessor` will stamp the log telemetry already